### PR TITLE
feat: Adding sdcore-config interface

### PR DESCRIPTION
**Pull request by gatici**
Adding sdcore-config interface which provides Webui GRPC address (Webui hostname + GRPC port)  to 5G control plane NF's.

### CODE REVIEW

**Review by ghislainbourgeois — CHANGES_REQUESTED** [review]

File: docs/json_schemas/dns_record/v0/requirer.json [hunk 1]
```diff
@@ -37,6 +37,9 @@
     "RecordClass": {
       "const": "IN",
       "description": "Represent the DNS record classes.",
+      "enum": [
+        "IN"
+      ],
```
[the remark this answers is not in the document]
Comment (gatici) [reply docs/json_schemas/dns_record/v0/requirer.json:42]: This is done automatically when we run `tox -e build-json-schemas`.  I will revert it.

File: interfaces/sdcore_config/v0/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# `sdcore-config`
+
+## Usage
+
+Within 5G, the Webui URL needs to be known by other components as SD-Core Webui Operator is responsible for the configuration of the SD-Core network.
```
[the remark this answers is not in the document]
Comment (gatici) [reply interfaces/sdcore_config/v0/README.md]: Done

File: interfaces/sdcore_config/v0/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# `sdcore-config`
+
+## Usage
+
+Within 5G, the Webui URL needs to be known by other components as SD-Core Webui Operator is responsible for the configuration of the SD-Core network. 
+
+Hence, 5G NF's need to connect to config server using Webui URL which is consist of Webui Operator hostname which equals to application name and a fixed GRPC port.
```
[the remark this answers is not in the document]
Comment (gatici) [reply interfaces/sdcore_config/v0/README.md]: Done

File: docs/json_schemas/dns_record/v0/requirer.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ghislainbourgeois) [reply docs/json_schemas/dns_record/v0/requirer.json:42]: Ok, understood.

**Review by ghislainbourgeois — APPROVED** [review]

**Review by markbeierl — APPROVED** [review]

**Review by tonyandrewmeyer — COMMENTED** [review]
This could probably get added to the Telco table in the top-level README, I think? Presumably with draft since there aren't tests yet.

**Comment by gatici** [comment]
> This could probably get added to the Telco table in the top-level README, I think? Presumably with draft since there aren't tests yet.

Hello @tonyandrewmeyer, The interface is added under Telco table and tests are added. 
Could you please review  again ?

**Review by tonyandrewmeyer — APPROVED** [review]
Thanks for adding those! Looks great to me.

**Comment by gatici** [comment]
> Thanks for adding those! Looks great to me.

Thank you @tonyandrewmeyer. Could you help me to merge it if everything seems fine ? Then, we can start to use it.

**Pull request merged**